src/player.cpp

```diff
@@ -3148,6 +3148,10 @@ void Player::postAddNotification(Thing* thing, const Cylinder* oldParent, int32_
 
 			for (const auto& it : openContainers) {
 				Container* container = it.second.container;
+				if (container == nullptr) {
```

Comment:
This check is unnecessary; pointers are not useful for validating whether a pointer points to valid memory. There is no instance where an openContainer is assigned a Container that is nullptr.